nexus/db-model/src/vmm.rs

```diff
@@ -43,6 +43,9 @@ pub struct Vmm {
     /// The IP address at which this VMM is serving the Propolis server API.
     pub propolis_ip: ipnetwork::IpNetwork,
 
+    /// The socket port on which this VMM is serving the Propolis server API.
+    pub propolis_port: i32,
```

Comment:
I'd look at using `nexus_db_model::SqlU16` for this--it has some helper impls that might cut down on the number of `u16`/`i32` conversions Nexus needs to do.

Reply:
oh! good to know, thanks